src/components/AddNewBookForm/_AddNewBookForm.scss

```diff
@@ -11,37 +11,34 @@ $abp-shadow: 0 3px 12px rgba(4, 5, 5, 0.5);
     align-content: center;
     position: fixed;
     z-index: 10;
+    left: 30%;
     top: 40%;
     background: black;
-    width: 300px;
-    height: 150px;
+    width: 40%;
```

Comment:
It seems like you are just reverting the changes that I made.

Did you check this on mobile and tablet size?

I quickly changed it last night at 00.30h because it looked horrible on mobile and tablet.

Reply:
Before, it always shows on the center of screen.  Your change made it at somewhere on the left part of screen and the position will change on different screen size.
And too much small fonts shown in 3secs is never a good UX.

And yes, you should check how it looked like before.

Reply:
And on desktop.

Reply:
I'm sorry that I didn't check it on mobile for my version. But you should also check yours.

I'm really tired of arguing all about this stuff these days. UX is not what you think yourself likes, but a balance for different people. Please be aware that all the changes I made were because I found a problem in it which you guys didn't aware. I'm not a person who likes to talk too much. I'd rather just do it. 

Every time you find me did something not good enough which I might not intentionally do. You'd always criticize it. During the time, I've also found a lot of problems that you/or others made but I chose not to raise an argument, instead, I tried to fix them since I don't see any value in reckless criticizing upon others. And, I've also spent days and hours chasing your bugs and fix back damages what you modification upon my code did... I'm not happy to say this and didn't plan to say this ever. However...  This is bad. I'm very SORRY.